examples/gltf/app.js

```diff
@@ -41,12 +41,15 @@ const DEFAULT_OPTIONS = {
 };
 
 function loadGLTF(urlOrPromise, gl, options = DEFAULT_OPTIONS) {
-  const promise = urlOrPromise instanceof Promise ? urlOrPromise : window.fetch(urlOrPromise).then(res => res.arrayBuffer());
+  const promise = urlOrPromise instanceof Promise
+   ? urlOrPromise
+   : window.fetch(urlOrPromise).then(res => urlOrPromise.endsWith('.gltf') ? res.json() : res.arrayBuffer());
+  let gltfParser;
 
   return promise.then(data => {
-    const gltfParser = new GLTFParser();
-    const gltf = gltfParser.parse(data);
-
+    gltfParser = new GLTFParser({uri: urlOrPromise});
```

Comment:
Hmm, I guess this could have been nicer as an option to `parse()` rather than the constructor. For next revision of loaders.gl...